Automattic/pocket-casts-ios
# File downloads use switch instead of check box for include starred

| 📘 Part of: #2430  |  
|:---:|

Fixes # 

| 1 | 2 |
| - | - |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-11-18 at 22 56 40](https://github.com/user-attachments/assets/8816a5b6-7a6b-4f14-8372-1bea4aea6d56) | ![Simulator Screenshot - iPhone 16 Pro - 2024-11-18 at 23 04 46](https://github.com/user-attachments/assets/11190734-f71d-4a6a-a614-5b053d3b1677) |


This PR just does a small change in terms of UI on the file downloads screen so that including starred episodes is a switch instead of looking a different category of files. This matches the UI on Android.

I also used this chance to refactor the code from a `if` to `switch`

## To test

- Start the app
- Download some episodes and star some of them
- Go Profile -> Downloads -> Clean Up
- Check if the UI match the screen shot above
- Select some types of files to clean
- Check and uncheck the starred switch and see the changes

## Checklist
